t/testhelpers.sh

```diff
@@ -808,3 +808,10 @@ setup_pure_ssh() {
       exit 0
   fi
 }
+
+ssh_remote() {
+  local reponame="$1"
+  local destination=$(urlify "$(canonical_path "$REMOTEDIR/$reponame.git")")
+  [ -z "${destination##/*}" ] || destination="/$destination"
```

Comment:
Yes, I believe those are functionally equivalent.  My goal is to prepend a slash iff it lacks one.  This is required for Windows, but things should work just fine on Unix without it.  I'll add a comment.

Also, having worked on Git, I've learned that invoking processes in shell on Windows is just eternally slow because Windows lacks a native `fork` primitive.  (Running a Unix shell under either WSL 1 or 2 is much faster as a result.)  So it's been drilled into my head to use built-in shell primitives whenever possible, since they perform much better there.